tailwind.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{js,jsx,ts,tsx}"],
+  content: ["./index.html", "./src/**/*.{js,jsx,ts,tsx}"],
```

Comment:
nit: we're likely not going to use any Tailwind classes in `index.html` do we? That file is mostly a placeholder for our main script, so I'm not sure having `index.html` here is necessary.